### PR TITLE
[WIP] Add kerr null-geodesics

### PR DESCRIPTION
**Pull request by ritzvik**
Referring to the papers
https://arxiv.org/pdf/1601.02063.pdf
http://www.math.mcgill.ca/gantumur/math599w17/project-kerr.pdf

### CODE REVIEW

File: src/einsteinpy/metric/kerr_null.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# paper : https://arxiv.org/pdf/1601.02063.pdf
+# paper : http://www.math.mcgill.ca/gantumur/math599w17/project-kerr.pdf
+
+_c = constant.c.value
+_G = constant.G.value
+
+
+class KerrNull:
+    """
+    Class for defining methods for Kerr Null Geodesics (Schwarzschild Null Geodesics when J=0)
```
Comment (shreyasbapat) [inline src/einsteinpy/metric/kerr_null.py]: My proposal, how about we define a class explicitly, which inherits this class. Which is Schwarzschild Null?

Comment (ritzvik) [reply src/einsteinpy/metric/kerr_null.py]: Yes. It's a good idea.
But let's first write KerrNull

**Comment by shreyasbapat** [comment]
Reminder: Just 2 Days for release of 0.1.0

**Comment by ritzvik** [comment]
Some related repo : 
https://github.com/grtrace/grtrace

**Pull request closed without merging**